instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraSqlAttributesGetter.java

```diff
@@ -22,28 +23,33 @@ public String getDbSystem(CassandraRequest request) {
     return DbIncubatingAttributes.DbSystemIncubatingValues.CASSANDRA;
   }
 
-  @Deprecated
   @Override
   @Nullable
-  public String getUser(CassandraRequest request) {
-    return null;
+  public String getDbNamespace(CassandraRequest request) {
+    return request.getSession().getLoggedKeyspace();
   }
 
   @Override
-  @Nullable
-  public String getDbNamespace(CassandraRequest request) {
-    return request.getSession().getLoggedKeyspace();
+  public Collection<String> getRawQueryTexts(CassandraRequest request) {
+    return singleton(request.getQueryText());
   }
 
-  @Deprecated
+  @Nullable
   @Override
+  public String getServerAddress(CassandraRequest request) {
+    return null;
+  }
+
   @Nullable
-  public String getConnectionString(CassandraRequest request) {
+  @Override
+  public Integer getServerPort(CassandraRequest request) {
     return null;
   }
```

Comment:
why implement getServerAddress and getServerPort when getNetworkPeerInetSocketAddress is implemented below?